callback_plugins/ansible_version_check.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding:utf-8 -*-
+# https://adamj.eu/tech/2016/08/19/restricting-the-ansible-version-in-use/
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sys
+
+import ansible
+
+try:
+    # Version 2.0+
+    from ansible.plugins.callback import CallbackBase
+except ImportError:
+    CallbackBase = object
+
+
+def print_red_bold(text):
```

Comment:
In looking through trying to implement this I've found that the more we rely on Ansible, while trying to check the current Ansible version, we would break on older versions of Ansible.  In this instance `C.COLOR_ERROR` does not exist in Ansible 2.0.0.2 and bombs. :(  So, it seems we should very lightly rely on Ansible dependencies.  Currently I'm using `__version__` which has been around since the beginning.  Thoughts?